pkg/virtual/framework/dynamic/apiserver/apiserver.go

```diff
@@ -177,14 +180,21 @@ func (c completedConfig) New(virtualWorkspaceName string, delegationTarget gener
 
 	s.GenericAPIServer.Handler.GoRestfulContainer.Add(discovery.NewLegacyRootAPIHandler(c.GenericConfig.DiscoveryAddresses, s.GenericAPIServer.Serializer, "/api").WebService())
 
+	if !c.GenericConfig.SkipOpenAPIInstallation {
```

Comment:
Just for my understanding here to match it with the PR description: Your intention was that developers of virtual workspaces that use the `virtual` framework can disable OpenAPI, not that kcp admins can disable OpenAPI on their default kcp VWs, right?

Reply:
To put it the other way around, what do we actually want and/or need more? My understanding of this implementation is that it's more towards those who run virtual workspaces, i.e. kcp-admins. Do we want both options?

Reply:
I would say for now we should leave it to the implementors, not the admins. If someone wants to make it optional in their virtual workspace, they can add a cmd flag exposing this setting.